ajax/agent.php

```diff
@@ -40,9 +40,9 @@
 if (isset($_POST['action']) && isset($_POST['id'])) {
     $agent = new Agent();
     if (!$agent->getFromDB($_POST['id'])) {
-        Response::sendError(404, 'Unable to load agent #' . $_POST['id']);
-
-        return;
+        $exception = new \Glpi\Exception\Http\HttpException(404); // @phpstan-ignore-line
+        $exception->setMessageToDisplay('Unable to load agent #' . $_POST['id']);
+        throw $exception; // @phpstan-ignore-line
```

Comment:
Maybe something like this?:
```suggestion
        throw new \Glpi\Exception\Http\NotFoundHttpException('Unable to load agent #' . $_POST['id']);
```